force-app/main/default/lwc/nksKnowledgeViewCounter/nksKnowledgeViewCounter.js

```diff
@@ -0,0 +1,35 @@
+import { LightningElement, api, wire } from 'lwc';
+import countViews from '@salesforce/apex/NKS_AuditLogController.countViews';
+import isKnowledgeUser from '@salesforce/apex/NKS_AuditLogController.isKnowledgeUser';
+
+export default class NksKnowledgeViewCounter extends LightningElement {
+    @api recordId;
+    siteURL;
+    numOfViews = 0;
+    hasPermission = false;
+
+    connectedCallback() {
+        this.siteURL = '/apex/Audit_Log_Knowledge?Id=' + this.recordId;
+    }
+
+    @wire(isKnowledgeUser)
+    wiredRes({ error, data }) {
```

Comment:
Denne kan vel strengt tatt hete wiredIsKnowledgeUser for å følge navnekonvensjonen under der den heter wiredCountViews 🤓